.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\aungm\\AppData\\Local\\Programs\\Python\\Python38\\python.exe"
+}
```

Comment:
Please add `*.vscode` to _.gitignore_ file and remove this **.vscode** folder.

Reply:
Will do it.